examples/ota-requestor-app/linux/main.cpp

```diff
@@ -89,7 +92,9 @@ OptionSet cmdLineOptions = { HandleOptions, cmdLineOptionsDef, "PROGRAM OPTIONS"
                              "        advertisements. If none is specified, default value is 3840.\n"
                              "  -q/--delayQuery <Time in seconds>\n"
                              "        From boot up, the amount of time to wait before triggering the QueryImage\n"
-                             "        command. If none or zero is supplied, QueryImage will not be triggered.\n" };
+                             "        command. If none or zero is supplied, QueryImage will not be triggered.\n"
+                             "  -c/--RequestorCanConsent\n"
+                             "        If provided, the RequestorCanConsent field of the Query Image request is set to true.\n" };
```

Comment:
nit: I would probably be clear that in it's absence, the `RequestorCanConsent` field will be false